recipes/Text_to_Shell/Text_to_Shell.ipynb

```diff
@@ -47,35 +35,47 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "!pip install ollama"
+    "!pip install git+https://github.com/ibm-granite-community/utils"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "### Select a model\n",
     "\n",
-    "## One-shot Prompt with Granite Code 3b\n",
-    "\n",
-    "In One-shot prompting, you provide the model with a question and no examples. The model will generate an answer given its training. Larger models tend to do better at this task.\n",
+    "Select a Granite Code model from the [`ibm-granite`](https://replicate.com/ibm-granite) org on Replicate. Here we use the Replicate Langchain client to connect to the model.\n",
     "\n",
-    "Use the [ollama-python package](https://github.com/ollama/ollama-python) to access the model."
+    "To connect to a model on a provider other than Replicate, substitute this code cell with one from [this LLM component recipe](https://github.com/ibm-granite-community/utils/blob/main/recipes/Components/Langchain_LLMs.ipynb)."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 2,
```

Comment:
@fayvor I see the output is clear, but I thought this meant it still needs to be 'more clear' ? I don't actually know.

Reply:
Interesting. I'm not sure how to clear the execution count, but I can look into it. VSCode leaves these execution counts in place when clearing the outputs.

Reply:
Ok, so... clearing all outputs works to clear the `execution_count` of the cells -- if you are on a fresh kernel. So to clear the execution_count, you need to restart the kernel, then "Clear Outputs of all Cells".

Unfortunately, this doesn't work on VSCode, since the Jupyter editor doesn't let you "Clear All Outputs" if no outputs are visible, even if the `execution_count` is non-null.

Reply:
That said, I've cleared the execution_counts for this PR by opening the recipe using `jupyter notebook`.